README.md

```diff
@@ -78,6 +78,7 @@ to be triggered on `GoyoEnter` and `GoyoLeave` events.
 ```vim
 function! s:goyo_enter()
   silent !tmux set status off
+  silent !tmux resize-pane -Z
```

Comment:
How about if we first check whether if the pane is already zoomed? This should do the trick:

```vim
silent !tmux list-panes -F '\#F' | grep -q Z || tmux resize-pane -Z
```